src/main/java/com/urswolfer/gerrit/client/rest/http/groups/GroupsRestClient.java

```diff
@@ -144,11 +144,16 @@ public QueryRequest query() {
         };
     }
 
+    /**
+     * this method may does not support Gerrit versions lower than 3.2.0
+     * @param queryRequest {@link QueryRequest}
+     * @return {@link List<GroupInfo>}
+     * @throws {@link RestApiException}
```

Comment:
Can you please remove these docs? they do not provide any additional value in my opinion.

Reply:
done